commercetools-models/src/main/java/io/sphere/sdk/stores/StoreDraft.java

```diff
@@ -32,8 +34,11 @@
     @Nullable
     List<ResourceIdentifier<Channel>> getSupplyChannels();
 
+    @Nullable
+    CustomFieldsDraft getCustom();
+
     static StoreDraftDsl of(final String key, @Nullable final LocalizedString name) {
-        return new StoreDraftDsl(null, key, new ArrayList<>(), name, null);
+        return new StoreDraftDsl(null, null, key, new ArrayList<>(), null, null);
```

Comment:
why is the name gone?

Reply:
after the rebuilt and the regeneration of the code it was messed up I'll re-put the name there